applications/giftless/templates/deployment.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: apps/v1
```

Comment:
Two almost-identical copies of a long resource like this make it hard to understand what the differences are between them. I think it's just which config map is mounted? It looks like fixing this in Helm is tricky because you have to understand variable scopes extremely well, so I'm good with leaving that for a later day, but could you at least add a Helm comment saying what's different between the two copies?

Reply:
Done.